osimpipeline/study.py

```diff
@@ -125,7 +125,7 @@ def list_condition_names():
         self.marker_trajectories_fpath = os.path.join(
                 self.expdata_path, 'marker_trajectories.trc')
         self.ground_reaction_fpath = os.path.join(
-                self.expdata_path, 'ground_reaction_orig.mot')
+                self.expdata_path, 'ground_reaction.mot')
```

Comment:
I haven't figured out exactly how we should deal with this yet. It's probably worth coming back to in another PR, since, other than reverting the change, addressing this seems like it could be a little involved.

Reply:
Sounds good.